### PR TITLE
Azure function

### PR DESCRIPTION
**Pull request by rtdickey**
Was able to return a message on the home page from a fetch to BoardGameAPI

### CODE REVIEW

**Comment by rtdickey** [comment]
Found error in setup. Closing for now.

**Comment by rtdickey** [comment]
The overall issue was two-fold. First, the example out of the box was not returning json. It was returning a plain string.
Second, You have to go into the api folder and do npm start to build any updates to the functions. Then you can run swa start --api-location api

File: api/BoardGameAPI/function.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
```
Comment (kevinrs5855) [inline api/BoardGameAPI/function.json:4]: We might need to change this in the future

File: api/BoardGameAPI/function.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get",
```
Comment (kevinrs5855) [inline api/BoardGameAPI/function.json:9]: I think we just need get not post

**Review by kevinrs5855 — APPROVED** [review]

**Pull request merged**